tests/rptest/tests/debug_bundle_test.py

```diff
@@ -162,6 +162,29 @@ def test_post_debug_bundle(self, ignore_none: bool):
         assert res.headers['Content-Type'] == 'application/json', res.json()
         assert not node.account.exists(file)
 
+        # Get the non-existant debug bundle
+        self._assert_http_error(requests.codes.conflict,
+                                self.admin.get_debug_bundle,
+                                node=node)
+
+        # Cancel the non-exstant debug bundle
```

Comment:
should it be

```suggestion
        # Delete the non-exstant debug bundle
```

Reply:
That's how I wrote it first, but it's really a cancel of the running process, so this is more descriptive than the request method.

Reply:
> fair enough! looks like a typo 😇

Well, there is another typo there :disappointed:

Reply:
🤫